client/src/reducers/teams.js

```diff
@@ -0,0 +1,22 @@
+import { GET_TEAM_MEMBERS } from '../constants/actionTypes';
+
+const initialState = {
+  currentTeam: {
+    members: [],
+  },
+};
+
+const teamsReducer = (state = initialState, action) => {
+  switch (action.type) {
+    case GET_TEAM_MEMBERS:
+      return {
+        ...state,
+        currentTeam: {
```

Comment:
I think here you are replacing the whole `currentTeam` object here

I'd suggest to just replace members
```suggestion
        currentTeam: {
          ...state.currentTeam
```